global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "7.0.202",
+    "version": "7.0.203",
```

Comment:
@vzarytovskii Could it be possible to keep an older version (given we have `rollForward` enabled), so it's not required for FCS contributors to immediately install `7.0.203`?

Reply:
We almost always want to be on the latest stable one, and switching to the previews closer to release, around summer, when we want to "dogfood" new .NET. And we want to build with it, run with it, etc. I don't think that we can do it properly in CI with just rollForward, we need to specify exact versions to install.